t.js

```diff
@@ -0,0 +1,32 @@
+while (!!foo1) {
+}
+
+// should not report no-undef and no-unused-vars
+// should report no-empty
```

Comment:
Redundant double negation.

```suggestion
while (foo1) {
```